main.c

```diff
@@ -80,19 +80,19 @@ static void fatalError(const enum fatal_error type)
         taskDISABLE_INTERRUPTS();
 
         for(;;) {
-                LED_enable(1);
-                LED_enable(2);
+                led_enable(LED_GPS);
```

Comment:
You are talking something like a panic function eh.  Not unlike what Linux does.  That would be a wise idea as I have had need for this in code.